lib/internal/test_runner/tests_stream.js

```diff
@@ -71,10 +71,12 @@ class TestsStream extends Readable {
   }
 
   enqueue(nesting, loc, name) {
+    const file = loc.file.startsWith('file://') ? loc.file.substring(7) : loc.file;
```

Comment:
please use `fileURLToPath`. 
also - we should handle other events besides enqueue. the correct place for the fix would probably be https://github.com/nodejs/node/blob/d755f1ae88d37f3779eb50f80b62daffcd412e89/lib/internal/test_runner/test.js#L355-L360

Reply:
I applied this thank you